homeassistant/components/songpal/media_player.py

```diff
@@ -75,9 +80,14 @@ async def async_setup_entry(
     async_add_entities: AddEntitiesCallback,
 ) -> None:
     """Set up songpal media player."""
+    if SONGPALDATA not in hass.data:
+        hass.data[SONGPALDATA] = []
     name = config_entry.data[CONF_NAME]
+    entity_id = "media_player."+name.lower().replace(" ","_")  # What is the proper way to get the entity id?
```

Comment:
This is probably not the way to do this, but don't know how this works. 
Any help appreciated

Reply:
The entity_id of the entity is available inside the entity as `self.entity_id`.